easybuild/easyconfigs/h/HPX/HPX-1.10.0-foss-2024a.eb

```diff
@@ -0,0 +1,58 @@
+# #
+# Author:    Benjamin Czaja (benjamin.czaja@surf.nl)
+# Institute: SURF(sara)
+#
+# #
+easyblock = 'CMakeNinja'
+
+name = 'HPX'
+version = '1.10.0'
+
+homepage = 'http://stellar-group.org/libraries/hpx/'
+description = """HPX (High Performance ParalleX) is a general purpose C++ runtime system
+ for parallel and distributed applications of any scale."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/STEllAR-GROUP/%(namelower)s/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['5720ed7d2460fa0b57bd8cb74fa4f70593fe8675463897678160340526ec3c19']
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    ('Ninja', '1.12.1'),
+    ('pkgconf', '2.2.0'),
+]
+dependencies = [
+    ('HDF5', '1.14.5'),
+    ('Boost', '1.85.0'),
+    ('hwloc', '2.10.0'),
+    ('gperftools', '2.16'),
+]
+
+build_type = 'Release'
+
+configopts = '-DCMAKE_CXX_COMPILER=g++ '
+configopts += '-DHPX_WITH_MALLOC=tcmalloc '
+configopts += '-DHPX_WITH_HWLOC=TRUE '
+configopts += '-DHPX_WITH_GOOGLE_PERFTOOLS=TRUE '
+configopts += '-DHPX_WITH_NETWORKING=TRUE '
+configopts += '-DHPX_WITH_PARCELPORT_TCP=FALSE '
+configopts += '-DHPX_WITH_PARCELPORT_MPI=TRUE '
+configopts += '-DHPX_WITH_TESTS=FALSE '
+configopts += '-DHPX_WITH_EXAMPLES=TRUE '
+# configopts += '-DHPX_WITH_MAX_CPU_COUNT=128' #this should be handled by a hook for the system
+# configopts += '-DHPX_WITH_MAX_CPU_COUNT=' + os.cpu_count()
+configopts += '-DHPX_WITH_FETCH_ASIO=TRUE '
+
+bin_lib_subdirs = ['lib/%(namelower)s/']
```

Comment:
remove unused param
```suggestion
bin_lib_subdirs = ['lib/%(namelower)s/']
```

Reply:
`bin_lib_subdirs` is needed to make sure that relevant path is added to `LD_LIBRARY_PATH` before the sanity check